receiver/otlpreceiver/README.md

```diff
@@ -59,7 +59,7 @@ receivers:
   otlp:
     protocols:
       http:
-        endpoint: "localhost:55681"
+        endpoint: "localhost:4318"
```

Comment:
We should not have a default yet for this correct? Until you merge then I will push my PR to share same port

Reply:
OK, I'll revert http port changes, but will keep gRPC port changes.

Reply:
Done.